sys/cmake/README.md

```diff
@@ -0,0 +1,24 @@
+== BlocksDS CMake
```

Comment:
If you're writing a markdown file, you need to do `# BlocksDS CMake` and `## Usage` so that the headers are parsed correctly.